src/praisonai/praisonai/cli/commands/config.py

```
@@ -56,18 +70,28 @@ def config_get(
+        for k in keys:
+            if isinstance(value, dict) and k in value:
+                value = value[k]
+            else:
+                output.print_error(f"Key not found: {key}")
+                raise typer.Exit(1)
+        
+        if output.is_json_mode:
+            output.print_json({"key": key, "value": value})
+        else:
+            output.print(f"{key} = {value}")
+    except Exception as e:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)